docs/src/main/paradox/jdbc.md

```diff
@@ -1,8 +1,225 @@
-# Offset in relational DB with JDBC
+# Offset in a relational DB with JDBC
 
-Not implemented yet, see [issue #23](https://github.com/akka/akka-projection/issues/23).
+The @apidoc[JdbcProjection] has support for storing the offset in a relational database using
+JDBC.
+
+The source of the envelopes can be @ref:[events from Akka Persistence](eventsourced.md) or any other `SourceProvider`
+with supported @ref:[offset types](#offset-types).
+
+Different from the common envelope handler, the @apidoc[JdbcHandler] receives a @apidoc[JdbcSession] instance and an envelope.   
```

Comment:
I don't think we should assume that the user knows about "common envelope handler". This sentence can be skipped here, we'll get to the details further down.